deploy/bicep/modules/appinsights.credentials.to.keyvault.bicep

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+param environmentName string
+param applicationInsightsName string
+param keyVaultName string
+param keyVaultResourceGroup string = resourceGroup().name
+param connectionStringSecretName string = 'AppInsightsConnectionString'
+param utcValue string = utcNow()
+
+resource applicationInsights 'Microsoft.Insights/components@2020-02-02-preview' existing = {
+  name: applicationInsightsName
+} 
+
+module acrLoginServerNameSecret './akv.secrets.bicep' = {
+  name: 'appinsights-connstr-${utcValue}'
```

Comment:
If the template were to be re-applied, we'd create another deployment here. That might be ok--if so then doesn't it also need to be done for appinsightsSecrets in main.bicep above? Seems like we want the deployment to be idempotent if possible, though so using a fixed name would be better if it's possible to do so.

Reply:
Removed the utcValue and instead added uniqueString() with subscription id.